git_machete/utils.py

```diff
@@ -94,7 +94,21 @@ def debug(msg: Optional[str] = None) -> None:
     if debug_mode:
         function_name = bold(inspect.stack()[1].function)
         args, _, _, values = inspect.getargvalues(inspect.stack()[1].frame)
-        function_args = bold(f"({', '.join([arg+'='+str(values[arg]) if arg != 'self' else 'self' for arg in args])})")
+
+        redact_arg_by_name = ['token']
+        for arg in redact_arg_by_name:
```

Comment:
Also, `s/redact/redacted`

Reply:
@PawelLipski why is this wrong? `redact argument by name` sounds correct to me

Reply:
Ok, make sense